### ISSUE TITLE
Whole stamp PDF is included (hidden) in the output PDF

### ISSUE THREAD

**Issue by Leseratte10** [heinrichreimer/latex-postage#6]
I just noticed that if I have a PDF file with multiple stamps, then the whole stamp PDF is included in the letter PDF generated by LaTeX. When I write a letter and include a stamp from a PDF that has multiple stamps on it, I can then open the PDF in a PDF viewer, select all text, and all the other information from the other stamps is present in there as well. The cropping is only done by the PDF viewer, not by LaTeX while generating the PDF.

Not that much of a problem if I just print that letter and send it to someone, but if I want to keep a digital copy of the letter as well there are two problems: 
- The file size is unnecessarily large because it includes all stamps, not just one
- Anyone who receives this PDF file from the letter will be able to extract all my other, not-yet-used stamps from the PDF and theoretically use them for their own letters. 

Is there a way this package can be modified so only the one stamp that is used on the letter is included in the PDF?

**Comment by janheinrichmerker** [comment]
Unfortunately, looking at the `graphicx` options, I can't find any option for "real" clipping of the included  PDF.
Do you know any other package for copying/including PDF in LaTeX?

**Comment by Leseratte10** [comment]
No, I don't know any other package in LaTeX that would have that functionality. I'm hoping someone else has an idea.
